### PR TITLE
Initial SPIRE 1.9.0 support

### PR DESCRIPTION
**Pull request by kfox1111**
Fixes: https://github.com/spiffe/helm-charts-hardened/issues/261

### CODE REVIEW

**Review by marcofranssen — APPROVED** [review]
LGTM 🚀  didn't actually test it, but the changes look very clean.

File: charts/spire/charts/spire-server/templates/configmap.yaml [hunk 1]
```diff
@@ -87,6 +87,12 @@ server:
   {{- end }}
 
 plugins:
+
+  {{- if .Values.credentialComposer.uniqueID.enabled }}
+  CredentialComposer:
```
Comment (faisal-memon) [inline charts/spire/charts/spire-server/templates/configmap.yaml:92]: Is this extra white space intentional?

Comment (kfox1111) [reply charts/spire/charts/spire-server/templates/configmap.yaml:92]: Yeah. CredentialComposer goes under the plugins section at like 89.

File: charts/spire/charts/spire-server/templates/configmap.yaml [hunk 2]
```diff
@@ -87,6 +87,12 @@ server:
   {{- end }}
 
 plugins:
+
+  {{- if .Values.credentialComposer.uniqueID.enabled }}
```
Comment (faisal-memon) [inline charts/spire/charts/spire-server/templates/configmap.yaml:91]:
```suggestion
plugins:
  {{- if .Values.credentialComposer.uniqueID.enabled }}
```

**Review by faisal-memon — APPROVED** [review]

**Pull request merged**